specs/swagger.yml

```diff
@@ -1131,6 +1141,11 @@ definitions:
       fingerprint:
         type: string
         x-nullable: true
```

Comment:
I think we should add cost to the required fields for router updates. I think fingerprint should probably be there as well. That's it's not there now it probably an oversight. In general we have most fields required for update, since if you only want to update some things, you use patch. Only concern there is backwards compatibility, but that shouldn't be a big concern here, since this API isn't widely used.

@andrewpmartinez Thoughts?

Reply:
Will add cost to be required. Did you want me to go ahead and add fingerprint there as well?

Reply:
Yes, please